package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "publisher": "lokalise",
+  "publisher": "huacnlee",
   "name": "i18n-ally",
-  "version": "2.8.1",
+  "version": "2.8.4",
```

Comment:
Can you limit changes to the new engine, please?

I'll add the changelog and version during the release.

Reply:
Done